script/openqa-bootstrap

```diff
@@ -58,8 +58,13 @@ fi
 /usr/share/openqa/script/fetchneedles
 
 if ping -c1 gitlab.suse.de. ; then
+        sles_needles_giturl="https://gitlab.suse.de/openqa/os-autoinst-needles-sles.git"
+        sles_needles_directory="/var/lib/openqa/tests/opensuse/products/sle/needles"
 	# clone SLE needles if run from within SUSE network
-	[ -d /var/lib/openqa/tests/opensuse/products/sle/needles ] || git clone https://gitlab.suse.de/openqa/os-autoinst-needles-sles.git /var/lib/openqa/tests/opensuse/products/sle/needles
+	if [ -d $sles_needles_directory ]; then
```

Comment:
this reversed logic and actually broke initial clone. https://github.com/os-autoinst/openQA/pull/2262 fixes it

thx @perlpunk